roles/nginxplus/files/conf/http/abid-staging.conf

```diff
@@ -29,8 +29,8 @@ server {
     ssl_prefer_server_ciphers  on;
 
     location / {
-        app_protect_enable on;
-        app_protect_security_log_enable on;
+#        app_protect_enable on;
```

Comment:
can these be deleted?  Is there a reason for leaving them?

Reply:
Heya @carolyncole we are leaving them off until we get training from f5/nginx on how to set them up. I suppose we can delete them and enable them after the training but I elected to add them.